modules/registry/registry-cpi/src/main/java/org/apache/airavata/registry/cpi/ComputeResource.java

```diff
@@ -231,6 +231,12 @@
      */
     void removeComputeResource (String resourceId) throws AppCatalogException;
 
+    /**
+     * This method will remove given resource from the system
```

Comment:
Comment here should be updated to reflect that the method will undelete the resource

Reply:
@machristie Done